### PR TITLE
bump upstream collector version to 0.66

### PR DESCRIPTION
**Pull request by srikanthccv**
Need couple of fixes from the latest release in another PR.

### CODE REVIEW

**Comment by makeavish** [comment]
@srikanthccv When are we planning to ship this?

**Comment by srikanthccv** [comment]
Next release unless there is something major blocking.

**Comment by srikanthccv** [comment]
To be precise, I didn't mean 0.11.4. It doesn't need any of the changes from this. I meant for 0.12 or later.

File: processor/signozspanmetricsprocessor/processor.go [hunk 1]
```diff
@@ -734,8 +733,8 @@ func (p *processorImp) buildDimensionKVs(serviceName string, span ptrace.Span, o
 	dims := pcommon.NewMap()
 	dims.PutStr(serviceNameKey, serviceName)
 	dims.PutStr(operationKey, span.Name())
-	dims.PutStr(spanKindKey, span.Kind().String())
-	dims.PutStr(statusCodeKey, span.Status().Code().String())
+	dims.PutStr(spanKindKey, "SPAN_KIND_"+strings.ToUpper(span.Kind().String()))
```
Comment (nityanandagohain) [inline processor/signozspanmetricsprocessor/processor.go:736]: Is this for backward compatibility?

Comment (srikanthccv) [reply processor/signozspanmetricsprocessor/processor.go:736]: Correct. Otherwise, alerts/dashboards break.

Comment (nityanandagohain) [reply processor/signozspanmetricsprocessor/processor.go:736]: Cool.

**Review by nityanandagohain — APPROVED** [review]

**Review by makeavish — APPROVED** [review]
LGTM.
@srikanthccv Have you tested clickhousetraceexporter too?

**Pull request merged**